azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -220,6 +220,29 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"client_certificate_common_name": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"certificate_common_name": {
```

Comment:
i think we can make this
```suggestion
						"common_name": {
```